zcash_client_sqlite/src/testing/pool.rs

```diff
@@ -1599,6 +1613,212 @@ pub(crate) fn fully_funded_fully_private<P0: ShieldedPoolTester, P1: ShieldedPoo
     );
 }
 
+#[cfg(feature = "orchard")]
+pub(crate) fn multi_pool_checkpoint<P0: ShieldedPoolTester, P1: ShieldedPoolTester>() {
```

Comment:
The PR description says it does not reproduce the bug; is that out-of-date? There appears to be uncovered code according to codecov.